lib/decorate.js

```diff
@@ -32,6 +34,41 @@ function decorate (instance, name, fn, dependencies) {
   }
 }
 
+function getDecorator (instance, name) {
+  if (checkExistence(instance, name)) {
+    return instance[name]
+  }
+
+  const prop = getProp(instance[kRequest], name) ?? getProp(instance[kReply], name)
+  if (!prop) {
+    throw new FST_ERR_DEC_UNDECLARED(name)
+  }
+
+  if (prop.value) {
+    return prop.value
+  }
+
+  return prop
+}
```

Comment:
Maybe we can't create a cache system with a key combining instance unique identifier with name.
Although it should not be a big performance issue because it is not meant to be use after start.

Reply:
It is also possible to create a get method by decorator type, `getRequestDecorators`, `getReplyDecorators`.